TooltipDecorator/tests/TooltipDecorator-specs.js

```diff
@@ -123,21 +127,222 @@ describe('TooltipDecorator Specs', () => {
+						<TooltipButton tooltipDelay={0} tooltipPosition="above" tooltipText={tooltipText}>Label</TooltipButton>
+					</FloatingLayerController>
+				);
+
+				const button = screen.getByRole('button');
+
+				button.getBoundingClientRect = jest.fn(() => {
+					return {
+						width: 300,
+						height: 300,
+						top: 600,
+						left: 600,
+						bottom: 0,
+						right: 0
+					};
+				});
+
+				act(() => button.focus());
+				fireEvent.mouseOver(button);
+
+				await waitFor(() => {
+					const tooltipArrow = screen.getByText('Tooltip').parentElement.parentElement;
+					const expected = 'tooltip above';
+
+					expect(tooltipArrow).toHaveClass(expected);
+				});
+			});
+
+			test('should have \'below\' className when tooltipPosition is set to \'below\'', async () => {
+				const tooltipText = 'Tooltip';
+				render(
+					<FloatingLayerController>
+						<TooltipButton tooltipDelay={0} tooltipPosition="below" tooltipText={tooltipText}>Label</TooltipButton>
+					</FloatingLayerController>
+				);
+
+				const button = screen.getByRole('button');
+
+				button.getBoundingClientRect = jest.fn(() => {
+					return {
+						width: 300,
+						height: 300,
+						top: 600,
+						left: 600,
+						bottom: 0,
+						right: 0
+					};
+				});
+
+				act(() => button.focus());
+				fireEvent.mouseOver(button);
+
+				await waitFor(() => {
+					const tooltipArrow = screen.getByText('Tooltip').parentElement.parentElement;
+
+					const expected = 'tooltip below';
+
+					expect(tooltipArrow).toHaveClass(expected);
+				});
+			});
+
+			test('should have \'left middle\' className when tooltipPosition is set to \'left middle\'', async () => {
+				const tooltipText = 'Tooltip';
+				render(
+					<FloatingLayerController>
+						<TooltipButton tooltipDelay={0} tooltipPosition="left middle" tooltipText={tooltipText}>Label</TooltipButton>
+					</FloatingLayerController>
+				);
+
+				const button = screen.getByRole('button');
+				act(() => button.focus());
+				fireEvent.mouseOver(button);
+
+				await waitFor(() => {
+					const tooltipArrow = screen.getByText('Tooltip').parentElement.parentElement;
+					const expected = 'tooltip left middleArrow';
+
+					expect(tooltipArrow).toHaveClass(expected);
+				});
+			});
+
+			test('should have \'right middle\' className when tooltipPosition is set to \'right middle\'', async () => {
+				const tooltipText = 'Tooltip';
+				render(
+					<FloatingLayerController>
+						<TooltipButton tooltipDelay={0} tooltipPosition="right middle" tooltipText={tooltipText}>Label</TooltipButton>
+					</FloatingLayerController>
+				);
+
+				const button = screen.getByRole('button');
+				act(() => button.focus());
+				fireEvent.mouseOver(button);
+
+				await waitFor(() => {
+					const tooltipArrow = screen.getByText('Tooltip').parentElement.parentElement;
+					const expected = 'tooltip right middleArrow';
+
+					expect(tooltipArrow).toHaveClass(expected);
+				});
+			});
+		});
+	});
```

Comment:
please add this extra test to increase codecov even more. It will cover lines 296-297


			test('should have \'above centerArrow\' className when tooltipPosition is set to \'left\'', async () => {
				const tooltipText = 'Tooltip';
				render(
					<FloatingLayerController>
						<TooltipButton tooltipDelay={0} tooltipPosition="left" tooltipText={tooltipText}>Label</TooltipButton>
					</FloatingLayerController>
				);

				const button = screen.getByRole('button');

				button.getBoundingClientRect = jest.fn(() => {
					return {
						width: 300,
						height: 300,
						top: 600,
						left: 600,
						bottom: 0,
						right: 0
					};
				});

				act(() => button.focus());
				fireEvent.mouseOver(button);

				await waitFor(() => {
					const tooltipArrow = screen.getByText('Tooltip').parentElement.parentElement;
					const expected = 'tooltip above centerArrow';

					expect(tooltipArrow).toHaveClass(expected);
				});
			});